src/cpl/share_esmf/laiStreamMod.F90

```diff
@@ -120,6 +124,9 @@ subroutine lai_init(bounds)
        write(iulog,'(a,a)' ) '  stream_fldFileName_lai = ',trim(stream_fldFileName_lai)
        write(iulog,'(a,a)' ) '  stream_meshfile_lai    = ',trim(stream_meshfile_lai)
        write(iulog,'(a,a)' ) '  lai_tintalgo           = ',trim(lai_tintalgo)
+       write(iulog,'(a,a)' ) '  lai_mapalgo            = ',trim(lai_mapalgo)
+       write(iulog,'(a,a)' ) '  lai_dtlimit            = ',trim(lai_dtlimit)
```

Comment:
I looked at one of the build fails (that I reported in the last post) and saw this error:
`NVFORTRAN-S-0074-Illegal number or type of arguments to trim - keyword argument string (/glade/work/slevis/git/latest_master/src/cpl/share_esmf/laiStreamMod.F90: 128)`

Reply:
Thanks, @olyson, I will try it.

Reply:
The python unit tests still fail. That's expected. Same error as before.

The python sys test that was failing got past the build error. Now it complains about missing files, which I think is a recurring issue that gets fixed if we wait long enough.

The aux_clm test-suite is in progress, but looks promising for now relative to the last time that I ran it.